library/junos_install_config

```diff
@@ -250,6 +261,9 @@ def _load_via_netconf(module):
         logging.error(msg)
         module.fail_json(msg=msg)
 
+    if args['ignore_warning'] in BOOLEANS + ["True", "False"]:
```

Comment:
@stacywsmith I have tested all possible scenarios and dont see any failure case. Can you help me identifying the condition in which we would require this?